doc/active_docs/account_management_api.json

```diff
@@ -1261,6 +1261,60 @@
             "content": {}
           }
         }
+      },
+      "post": {
+        "summary": "Service Subscription Create",
+        "description": "Create service contracts of an account",
+        "tags": ["Accounts"],
+        "parameters": [
+          {
+            "name": "account_id",
+            "in": "path",
+            "description": "ID of the account.",
+            "required": true,
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "service_id",
```

Comment:
1. It is not recommended (and probably just wrong) to have both query and body parameters in a POST request.
2. if we have the plan ID of the service plan, we don't actually need the service ID, because if would be inferred from the service plan.

Reply:
agreed with both points we can remove the service ID

Reply:
This PR need more changes I am working on it